README.md

```diff
@@ -109,10 +109,38 @@ In case your BEEFY keys are using the wrong cryptographic scheme, you will see a
 
 ## Running BEEFY
 
-Currently the easiest way to see BEEFY in action is to run a single dev node like so:
+Currently the easiest way to run BEEFY is to use a 3-node local testnet using `beefy-node`. We will call those nodes `Alice`, `Bob` and
+`Charlie`. Each node will use the built-in development account with the same name, i.e. node `Alice` will use the `Alice` development
+account and so on. Each of the three accounts has been configured as an initial authority at genesis. So, we are using three validators
+for our testnet.
+
+`Alice` is our bootnode is is started like so:
 
 ```
-$ RUST_LOG=beefy=trace ./target/debug/beefy-node --tmp --dev --alice --validator
+$ RUST_LOG=beefy=trace ./target/debug/beefy-node --alice -d /tmp/alice
 ```
 
-Expect additional (more useful) deployment options to be added soon.
+`Bob` is started like so:
+
+```
+RUST_LOG=beefy=trace ./target/debug/beefy-node --bob --port 30334 -d/ tmp/bob --bootnodes '/ip4/127.0.0.1/tcp/30333/p2p/ALICE_BOOTNODE_ID_HERE'
+```
+
+`Charlie` is started like so:
+
+```
+RUST_LOG=beefy=trace ./target/debug/beefy-node --charlie --port 30334 -d /tmp/charlie --bootnodes '/ip4/127.0.0.1/tcp/30333/p2p/ALICE_BOOTNODE_ID_HERE'
+```
+
+Note that all nodes are using a persistent DB located under `/tmp/xxx`. For `Bob` and `Charlie` this is not strictly necessary, however. The reason for this is the `ALICE_BOOTNODE_ID_HERE` placeholder.
```

Comment:
Is passing bootnodes really necessary if it's on the same machine? These node should discover each other easily, no?

Have you tried running just with `--tmp` and without `--bootnodes`?